modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -818,37 +824,51 @@
 
 	COOLDOWN_START(find_glass, remaining_heat, glassblowing_amount)
 	to_chat(user, span_notice("You finish heating up [blowing_item]."))
-	user.mind.adjust_experience(/datum/skill/smithing, 10) //creating an item gives you some experience, not a lot
-	user.mind.adjust_experience(/datum/skill/production, 5)
+	user.mind.adjust_experience(/datum/skill/smithing, 5)
```

Comment:
A closer look at skills related to making stuff might be needed, and especially with how skills are granted given how willy nilly these are just thrown around.